woocommerce/google-listings-and-ads
# Remove date from the notice box in request Review

### Changes proposed in this Pull Request:



Partially closes #1165 .

Remove the Fixed date in the BLOCKED status. Instead, we will show a new copy as discussed [here](pcTzPl-bx-p2#comment-560). 


### Detailed test instructions:


1. Connect your GLA account
2. You will need to have at least one issue. You can create one in the `wp_gla_merchant_issues` table
3. Go to Product feed `wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fproduct-feed`
4. You should see this 

<img width="1118" alt="Screenshot 2022-02-18 at 13 30 14" src="https://user-images.githubusercontent.com/5908855/154655812-34444ac5-c22f-458b-9ee5-5db4875e1745.png">


### Additional details:

- Removed ReqestReview helper since we dont need anymore those functions. 
- Adapting AccounReviewController and their tests
- Adapting the frontend to show the new copy instead of the date
